### PR TITLE
Clean up src/Form.php

### PR DESCRIPTION
**Pull request by machour**
Did the usual cleanups (single quotes, strict comparisons, short syntaxes, ..) and two small code refactoring (no bc break)

### CODE REVIEW

File: src/Form.php [hunk 1]
```diff
@@ -566,165 +575,132 @@ function print_ob_radio($objattr, $w, $h, $texto, $rtlalign, $k, $blockdir, $x,
...
 				$this->mpdf->Circle($cx, $cy, $radius * 0.4, 'DF');
 			}
 			$this->mpdf->SetFColor($this->colorConverter->convert(255, $this->mpdf->PDFAXwarnings));
 			$this->mpdf->SetDColor($this->colorConverter->convert(0, $this->mpdf->PDFAXwarnings));
 		}
 	}
 
-// In _putpages
-	function countPageForms($n, &$totaladdnum)
+	private function getCountItems($form)
```
Comment (machour) [inline src/Form.php]: not sure about this name.. better suggestion?

**Comment by finwe** [comment]
Running this code on example files (rebased on previous PRs) has failed with a Fatal error:

```
PHP Fatal error:  Uncaught Error: Call to undefined method Mpdf\Form::getCount() in /home/finwe/www/_github/mpdf/repo/src/Form.php:669
Stack trace:
#0 /home/finwe/www/_github/mpdf/repo/src/Mpdf.php(9650): Mpdf\Form->countPageForms(1, 0)
#1 /home/finwe/www/_github/mpdf/repo/src/Mpdf.php(11398): Mpdf\Mpdf->_putpages()
#2 /home/finwe/www/_github/mpdf/repo/src/Mpdf.php(2004): Mpdf\Mpdf->_enddoc()
#3 /home/finwe/www/_github/mpdf/repo/src/Mpdf.php(9238): Mpdf\Mpdf->Close()
#4 /home/finwe/www/_github/mpdf/repo-examples/example57_new_mPDF_v5-3_active_forms.php(216): Mpdf\Mpdf->Output()
#5 {main}
  thrown in /home/finwe/www/_github/mpdf/repo/src/Form.php on line 669
```

**Comment by machour** [comment]
@finwe Fixed along with another silly mistake in a constant name.

**Comment by finwe** [comment]
Thanks, could you please rebase to development instead of merging it?

**Comment by machour** [comment]
I'm trying but I think I'm miserably failing as I'm really not used to rebase.

I removed my last two commits (merge + constant name fix), then did `git push -f`, `git rebase development` `git pull && git push`.

Is the bunch of commits that were just added something that is supposed to happen? 
And I still need to re-commit the constant fix, but I'm holding off to avoid messing more things up.

Thanks for your help.

PS: If you rather do the rebase yourself to avoid wasting time, the last valid commit is 9986c9d and in line 1412, `self::RADIO` needs to be changed to `self::FLAG_RADIO`.

**Comment by finwe** [comment]
OK, I'll try myself in a bit.

**Comment by finwe** [comment]
About rebasing: I tried to summarize the recipe in a gist: https://gist.github.com/finwe/a95e26197be82f753686114720d42581

**Comment by machour** [comment]
Ran a diff on this PR and https://github.com/mpdf/mpdf/commit/efbf6ebe7d9ab6839f33e6c578ffd8ec09733218, only thing that changes in the constant name getting fixed. Neat 👌 

Thank you so much for the gist!

**Pull request closed without merging**